### DISCUSSION TITLE
push update to live app

### DISCUSSION THREAD

**Discussion by yairShiloach** [chatwoot/chatwoot#2340] [General]
Hi, we've upgraded to a new version,
The upgrade seems to have worked:
![Screen Shot 2021-05-26 at 9 20 21](https://user-images.githubusercontent.com/60548009/119611683-ad80a280-be03-11eb-9b93-ce529cac3683.png)

But I do not see that all the new features exist
For example it remains the same:
![Screen Shot 2021-05-26 at 9 21 52](https://user-images.githubusercontent.com/60548009/119611811-d739c980-be03-11eb-8e7e-c4ab3bd7cd5f.png)

**Comment by pranavrajs** [comment]
@yairShiloach What deployment method are you using? (Docker/Heroku/Linux VM)

**Reply by yairShiloach** [reply]
oh! Sorry I did not mention this:
Linux VM -  Ubuntu 20.04

**Comment by sojan-official** [comment]
could you try recompiling the assets? 
https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#if-precompile-fails
